boot/core/src/boot/task/built_in.clj

```diff
@@ -200,7 +200,14 @@
   - :type, :file, and :theme.
 
   The visual notification function will receive a map with four keys
-  - :title, :uid, :icon, and :message."
+  - :title, :uid, :icon, and :message.
+
+  The `notify` task always attempts to catch any exceptions that are
+  thrown so as not to cause your build to fail. If no notifications
+  are happening even if you specify `audible` or `visual`, you can see
+  any exceptions that are being caught by changing
+  `boot.util/*verbosity*` (at the command-line use: `boot -v` or `boot
+  -vv`)."
```

Comment:
How about logging a warning at default verbosity that there has been an exception? I like that this will never break a build but making exceptions invisible by default seems like it could lead to trouble.